wdl-lint/tests/lints/input-spacing/source.errors

```diff
@@ -0,0 +1,58 @@
+warning[CallInputSpacing]: input not properly spaced
+   ┌─ tests/lints/input-spacing/source.wdl:11:14
+   │
+11 │     call bar {
+   │              ^
+   │
+   = fix: input keyword must be separated from the opening brace ("{") by a single space
+
+warning[CallInputSpacing]: input keyword may not be preceded by a newline
+   ┌─ tests/lints/input-spacing/source.wdl:11:15
+   │  
+11 │       call bar {
+   │ ╭──────────────^
+12 │ │         input: a="something"
+   │ ╰────────^
+   │  
+   = fix: input keyword must be separated from the opening brace ("{") by a single space with no newlines
```

Comment:
This is the same[ish] span being reported twice. We only want one of the two diagnostics, and I'd prefer the second to the first.